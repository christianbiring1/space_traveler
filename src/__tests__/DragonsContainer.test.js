import React from 'react';
import renderer from 'react-test-renderer';
import DragonContainer from '../components/DragonsContainer';

it('matches Dragon container', () => {
  const tree = renderer.create(
    <DragonContainer />,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
