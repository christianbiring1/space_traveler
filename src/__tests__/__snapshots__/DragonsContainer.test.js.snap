// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches Dragon container 1`] = `
<div
  className="rocketContainer"
>
  <div
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    style={
      Object {
        "bottom": 16,
        "left": 16,
        "pointerEvents": "none",
        "position": "fixed",
        "right": 16,
        "top": 16,
        "zIndex": 9999,
      }
    }
  />
  <div
    className="imageContainer"
  >
    <img
      alt="1"
      className="image"
      src="https://image.com"
    />
  </div>
  <div
    className="rocketDetails"
  >
    <h3
      className="rocketName"
    >
      Dragon
    </h3>
    <p
      className="rocketDescription"
    >
      type:
       
      type
    </p>
    <p>
      Lorem ipsum dolor sit amet.
    </p>
    <button
      className="reserveRocketocketButton"
      onClick={[Function]}
      type="button"
    >
      Reserve Dragon
    </button>
  </div>
</div>
`;
